### DISCUSSION TITLE
Is it planned to update `wgpu.h` to use `WGPU_NULLABLE` like `webgpu.h`?

### DISCUSSION THREAD

**Discussion by laurelkeys** [gfx-rs/wgpu-native#346] [Ideas]
Are there plans on updating `wgpu.h` to use `WGPU_NULLABLE` like `webgpu.h` already does? This would be great for documenting `Option<>`al parameters, e.g.:
```cpp
//                                                          vvvvvvvvvvvvv
size_t wgpuInstanceEnumerateAdapters(WGPUInstance instance, WGPU_NULLABLE WGPUInstanceEnumerateAdapterOptions const * options
```

**Comment by rajveermalviya** [comment]
> Are there plans on updating wgpu.h to use WGPU_NULLABLE

yes, general idea in future is to generate our custom header using the same new generator used in [webgpu-headers](https://github.com/webgpu-native/webgpu-headers/pull/259).

though for now, you could send a PR for this one.
